### ISSUE TITLE
Should "activate env" error message suggest "conda activate" rather than "source activate"?

### ISSUE THREAD

**Issue by ceball** [conda/conda#6728]
```
$ conda --version
conda 4.4.7
$ activate myenv
Error: activate must be sourced. Run 'source activate envname'
instead of 'activate envname'.
```

**Comment by kalefranz** [comment]
We might end up getting rid of that error message altogether with conda 4.4.  I'm not sure.  Might be better just to add a deprecation warning to `activate` that people should be using `conda activate`.

**Comment by mbargull** [comment]
For the better or worse, we didn't add a deprecation warning, but this is ultimately resolved in `conda >=4.4.11` via #6669 by removing the executable permission on `activate`/`deactivate`.
